.github/workflows/ci.yml

```diff
@@ -80,14 +109,32 @@ jobs:
         uses: ./.github/actions/run-script
         with:
           script-name: "lint:scripts"
+
+  unit-test:
+    needs: lint-js
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the Repo
+      - name: Checkout
+        uses: actions/checkout@v3
+      # Setup Repo
+      - name: Setup Repo
+        uses: ./.github/actions/setup-repo
+        with:
+          node-version: 18
+      # Restore Packages from Cache
+      - name: Build Packages
+        uses: ./.github/actions/run-script
+        with:
+          script-name: "build"
       # Run Unit Tests
       - name: Unit Tests
         uses: ./.github/actions/run-script
         with:
           script-name: "test:ci"
 
   deploy-docs:
-    needs: build
+    needs: unit-test
```

Comment:
so to make sure I understand, we don't need 'build' because unit test runs a built itself? I think that's what I see above

Reply:
Unit tests previously ran as part of the `build` job. But now we're running again multiple node versions / OS combos, I wanted to avoid running the unit tests 4 times.

As a result, unit-tests relies on the build, and the docs-deploy will only occur once unit tests have passed 👍🏼